src/styles/ui/CardButton.scss

```diff
@@ -14,13 +14,22 @@
         transform: none;
       }
     }
+    &.simple {
+      cursor: default;
+      &:hover {
+        transform: none;
+      }
```

Comment:
Only for white cards that do nothing ;)

Reply:
yes the white cards in the hand don't do anything